src/components/dialogs/connectivity/connectivity-form-utils.js

```diff
@@ -39,13 +39,15 @@ export const getConnectivityPropertiesValidationSchema = () => {
 };
 
 export const getConnectivityWithPositionValidationSchema = (
-    id = CONNECTIVITY
+    isEquipmentModification = false
```

Comment:
this will break "getConnectivityWithPositionValidationSchema" usage in 
src/components/dialogs/network-modifications/line/characteristics-pane/line-characteristics-pane-utils.js